apps/dashboard/src/pages/_app.tsx

```diff
@@ -269,42 +269,31 @@ const ConsoleApp = memo(function ConsoleApp({
         transitionTimingFunction="ease"
```

Comment:
Only the changes done in _app.tsx are necessary changes. other changes are just for  reducing chakra dependencies